proto/src/lib.rs

```diff
@@ -1,8 +1,10 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
-pub use crate::prost::eraftpb;
+mod protos {
+    include!(concat!(env!("OUT_DIR"), "/protos/mod.rs"));
```

Comment:
Could we get this path from Jinkela? Otherwise it is a bit magic and keeping it up to date might be annoying

Reply:
Actually the path is given by build.rs. So that it does not have to be put inside build script or even the same directory. But I think your concern is correct, maybe we can output an environment variable in build script instead.